chicken-sloths/bangazon-site-sprint3
# Kb user customer

# Description
Combines Users and Customers so we log in and authenticate Customers, not Users.

## Problem to Solve
We need to combine Users and Customers.

## Steps to Test Solution
1. `npm run data:gen`
1. `npm run db:gen`
1. check database: `Customers` should be full and `Customers.password` should look like bullshit
